devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy

```diff
@@ -198,7 +198,8 @@ def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cm
 	if (cmd.notificationStatus == 0xFF) {
 		switch (cmd.event) {
 			case 0x09: //TAMPER
-				createEvent([name: "tamper", value: "detected"])
+				sendEvent(name: "tamper", value: "detected")
+				runIn(10, "clearTamper")
```

Comment:
check the `zwave-motion-sensor` dth. You'll need to put an `unschedule` of any old `clearTamper` calls here just in case tamper got triggered again.

Reply:
@greens I read about `unschedule` and this turns off my scheduling. In this case, I want to call `runIn` every time tamper is detected. 
Maybe to avoid `unschelude` I should use [overwrite: true]?

Reply:
I believe `[overwrite: true]` is the default behavior.

Reply:
Donald is correct. I guess that most places spell it out explicitly for clarity's sake.

@MGoralczykS the way it's done on other DTHs is to unschedule the old call and add a new one, so the delay is always 10s from the last tamper, but since overwriting happens by default anyway this doesn't seem to be necessary.